src/resources/members/controller.js

```diff
@@ -20,6 +20,102 @@ async function getAllMembers( req, res){
  }
 }
 
+async function addNewMember(req, res){ 
+
+    const memeber = req.body
+    console.log("memeber: ", memeber)
+
+    try { 
+
+        const newMember = await prisma.member.create({
+            data : { 
+                ...memeber
+            }
+        })
+
+        res.json({ theNewMemeber: newMember})
+
+    } catch(error) {
+
+        res.status(500).json({ error : error.message })
+    }
+}
+
+async function removeOneMemeber(req, res){ 
+
+    const id = req.params.id;
+    console.log("id: ", id)
+
+    try { 
+
+        const memeberToDelete = await prisma.member.delete({
+            where : {
+                id : parseInt(id)
+            }
+        })
+
+        res.json({ deleted : true })
+
+    } catch(error) {
+
+        res.status(500).json({ error : error.message})
+    }
+}
+
+async function updateOneMember(req, res){ 
+
+    const id = req.params.id;
+    const updatedData = req.body;
+    console.log("id: ", id, "\nupdatedData: ", updatedData)
+
+    try { 
+
+        const memeberNewDetails = await prisma.member.update({ 
+            where : {
+                id : parseInt(id)
+            }, 
+            data : { 
+                ...updatedData
+            }
+        })
+
+        res.json({ resp: memeberNewDetails})
```

Comment:
Let's try to have some consistency with responses. This one will end up looking like this to the frontend:

```
{
    resp: {
        // properties here
    }
}
```

I'd just pass the `memeberNewDetails` directly again; `res.json(memeberNewDetails)`